### PR TITLE
[SITES-287] Feature/summaries at top of pages

### PR DESCRIPTION
**Pull request by ltankey**
image

Add short_summary to meta description for SEO also.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -32,6 +32,7 @@ This file is influenced by http://keepachangelog.com/.
 - Changed Table of Contents heading to 'On this page'
 - News Articles get sorted by published_at date
 - News Articles short summary field can be edited
+- Short/long summary fields for all nodes sit beneath page abstract
```
Comment (tessereth) [inline CHANGELOG.md]: beneath page abstract? I thought they were the page abstract?

Comment (ltankey) [inline CHANGELOG.md]: Language fail - beneath for HAML - within from a markup perspective. Will modify for clarity.

File: app/views/templates/general_content.html.haml [hunk 2]
```diff
@@ -1,8 +1,17 @@
+- if node.short_summary
```
Comment (micapam) [inline app/views/templates/general_content.html.haml]: Is this not the case across the board? Could it go in application layout?

Comment (ltankey) [inline app/views/templates/general_content.html.haml]: It is, but some nodes may not have a defined short summary (i.e. SectionHome nodes).

With that in mind, would it still make sense for us to shift this into app layout?

File: app/views/application/_head.html.erb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 <head>
   <title><%= title %></title>
   <meta name="viewport" content="width=device-width, initial-scale=1">
+  <%= yield :meta_description %>
```
Comment (tessereth) [inline app/views/application/_head.html.erb]: Does this need an `if content_for? :meta_description`? I actually have no idea what happens if you yield to something that isn't defined.

File: app/views/templates/general_content.html.haml (hunk 2 above)
Comment (micapam) [inline app/views/templates/general_content.html.haml]: You are already checking to see if there is a short summary though aren't you? The conditional above.

Comment (ltankey) [inline app/views/templates/general_content.html.haml]: Good point - I'll move it into the application layout.

**Comment by ltankey** [comment]
@micapam I shifted the content_for block into the node layouts, which felt like a better place than their templates. I couldn't move them into the application layout as that makes an assumption that every page is represented by a node, which isn't necessarily true.

**Comment by micapam** [comment]
LGTM!

**Comment by tessereth** [comment]
LGTM

**Pull request merged**